docs/content/latest/api/ysql/datatypes/type_datetime/timezones/ways-to-spec-offset/_index.md

```diff
@@ -140,9 +140,9 @@ What? With the timezone set to _FooBar5_, the result of casting _2021-04-15 12:0
 
 ### Indirectly using a timezone name
 
-This is overwhelmingly the preferred approach because it's this, and only this, that brings you the beneficial automatic mapping to the _UTC offset_ value that reigns at the moment of execution of a sensitive operation according to the rules for Daylight Savings Time that the name keys to in the internal representation of the [_tz database_](https://en.wikipedia.org/wiki/Tz_database).  (See the section [Scenarios that are sensitive to the _UTC offset_ and possibly, additionally, to the timezone](../timezone-sensitive-operations/).) The names are automatically looked up in the _pg_timezone_names_ catalog view. See the section [Rules for resolving a string that's intended to identify a _UTC offset_](./name-res-rules/)
+This is overwhelmingly the preferred approach because it's this, and only this, that brings you the beneficial automatic mapping to the _UTC offset_ value that reigns at the moment of execution of a sensitive operation according to the rules for Daylight Savings Time that the name keys to in the internal representation of the [_tz database_](https://en.wikipedia.org/wiki/Tz_database).  (See the section [Scenarios that are sensitive to the _UTC offset_ or explicitly to the timezone](../timezone-sensitive-operations/).) The names are automatically looked up in the _pg_timezone_names_ catalog view. See the section [Rules for resolving a string that's intended to identify a _UTC offset_](./name-res-rules/)
 
 ### Indirectly using a timezone abbreviation
 
-Though this is possible, Yugabyte strongly recommends that you avoid this approach. The best that it can do for you is to bring you a fix value for the _UTC offset_ that's independent of the moment of lookup. But this is an exceedingly rare use case. If this is what you want, you can do it in a self-documenting way by specifying the _UTC offset_ [Directly as an interval value](#directly-as-an-interval-value)  as was explained above. Moreover, there are other risks brought by the attempt to use a timezone abbreviation to specify a _UTC offset_. See the section [Rules for resolving a string that's intended to identify a _UTC offset_](./name-res-rules/).
+Though this is possible, Yugabyte strongly recommends that you avoid this approach. The best that it can do for you is to bring you a fix value for the _UTC offset_ that's independent of the moment of lookup. But this is an exceedingly rare use case. If this is what you want, you can do it in a self-documenting way by specifying the _UTC offset_ [Directly as an _interval_ value](#directly-as-an-interval-value)  as was explained above. Moreover, there are other risks brought by the attempt to use a timezone abbreviation to specify a _UTC offset_. See the section [Rules for resolving a string that's intended to identify a _UTC offset_](./name-res-rules/).
```

Comment:
This is even stronger than "Yugabyte strongly recommends". Just flat-out "Don't do this."
```suggestion
Avoid this approach. The best that it can do for you is to bring you a fix value for the _UTC offset_ that's independent of the moment of lookup. But this is an exceedingly rare use case. If this is what you want, you can do it in a self-documenting way by specifying the _UTC offset_ [Directly as an _interval_ value](#directly-as-an-interval-value)  as was explained above. Moreover, there are other risks brought by the attempt to use a timezone abbreviation to specify a _UTC offset_. See the section [Rules for resolving a string that's intended to identify a _UTC offset_](./name-res-rules/).
```